sources/editor/Xenko.Core.Assets.Editor/Themes/ExpressionDark/TableViewScrollViewer.generic.xaml

```diff
@@ -15,157 +15,11 @@
   **********************************************************************************-->
 
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:xcdg="http://schemas.xceed.com/wpf/xaml/datagrid">
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
 
-  <ResourceDictionary.MergedDictionaries>
+  <!--<ResourceDictionary.MergedDictionaries>
```

Comment:
You can remove logic commented out, If we need to keep this for some reason mention why above that line.